### DISCUSSION TITLE
"Used by" list

### DISCUSSION THREAD

**Discussion by arkivanov** [arkivanov/MVIKotlin#90] [Show and tell]
It would be good to have a list of companies/users/projects where MVIKotlin is used. I suggest to stick to the following format, but feel free to omit/add/modify any details. ❤️

Project name: \<project name\>
Project type: production/pet/sample
Company name: personal/\<company name\>
Supported platforms: Android, JVM, Web, iOS, macOS, etc.
Link (an open source repository or a web site): \<link\>

**Comment by moffpage** [comment]
Project name: Vlife
Description: An ex super-app (more relevant) for motorists with loyalty programs, marketplace, payments and other services.
Company Name: LLC "GRAND ERA"
Project type: Production
Supported targets: currently only Android
You can install in in Google Play Store: https://play.google.com/store/apps/details?id=kz.vlife.grandera

**Comment by a-blekot** [comment]
Project name: Memorize shlokas (Sanskrit verses study)
Project type: production
Company name: personal (Aleksey Blekot)
Supported platforms: Android, iOS (ios not ready yet)
[PlayStore](http://play.google.com/store/apps/details?id=com.a_blekot.memorize_shloka)
[GitHub](https://github.com/a-blekot/memorize_shloka)

**Comment by t1r** [comment]
Project name: Compose Theme Builder
Project type: pet
Company name: personal
Supported platforms: Android, JVM, (iOS in future)
Link: [GitHub](https://github.com/t1r/ComposeThemeBuilder)

**Comment by MohamedRejeb** [comment]
Project name: Pokedex
Project type: sample
Company name: personal
Supported platforms: Android, JVM, iOS
Link: https://github.com/M0Coding/Pokedex

**Comment by CXwudi** [comment]
Project name: Realworld Conduit Example App
Project type: Sample, practice
Company name: Personal
Supported platforms: Android, JVM, Web
Link: https://github.com/CXwudi/realworld-cmp-decompose-mvikotlin-app

This is a frontend implementation of the [RealWorld](https://github.com/gothinkster/realworld) repository. The project was initially a practice to learn Compose Multiplatform development. Overall, I really enjoyed the development and the knowledge I gained along the way, especially regarding Decompose and MVIKotlin. Despite their complexity, they effectively address underlying problems that many developers, and even other navigation/MVI libraries, might not be aware of.

**Comment by 3moly** [comment]
<img width="100" height="100" alt="cedar" src="https://github.com/user-attachments/assets/06f55f06-ecb7-44bf-a177-4ff27d15db65" />

Project name: CedarJam - Note-Taking App
Project type: pet
Company name: personal
Supported platforms: Windows, macOS, Linux, Android, IOS
Link: https://github.com/3moly/CedarJam
